libs/gamify/applications/src/lib/models/application.model.ts

```diff
@@ -0,0 +1,8 @@
+export class ApplicationModel {
```

Comment:
Any specific reason why you don't want to name this `Application`?

Reply:
Nope, no particular reason